controllers/topology/internal/mergepatch/mergepatch.go

```diff
@@ -193,5 +194,9 @@ func (h *Helper) Patch(ctx context.Context) error {
 	if !h.HasChanges() {
 		return nil
 	}
+
+	log := ctrl.LoggerFrom(ctx)
+	log.V(4).Info("Patch", "Patch", string(h.patch))
```

Comment:
```suggestion
	log.V(10).Info("Patching object", "diff", string(h.patch))
```

This seems very verbose

Reply:
Verbosity >4 are almost impossible to use.
At level 5 we are logging all the predicates, and this adds a lot of noise
At level >=6 the client go roundtripper adds even more noise for each API call.

Also this log entry in this current form is simple at gets generated only when a object is actually patched, and this should not happen frequently